setup.py

```diff
@@ -110,6 +110,9 @@ def __init__(self, name, extra_cmake_args=[]):
 DEP_BUILD_DIR = os.path.join(PROJECT_DIR, 'build', 'deps')
 DEP_INSTALL_PATH = os.environ.get('AWS_C_INSTALL', os.path.join(DEP_BUILD_DIR, 'install'))
 
+if sys.platform != 'darwin' and sys.platform != 'win32':
+    AWS_LIBCRYPTO_INSTALL = os.environ.get('AWS_LIBCRYPTO_INSTALL', os.path.join(DEP_BUILD_DIR, 'libcrypto'))
```

Comment:
trivial/style: let's set `AWS_LIBCRTYPO_INSTALL = None` in an else-statement, or before the if-statement.

then later we can just `if AWS_LIBCRYPTO_INSTALL:` instead of repeating this complex if-statement again

Reply:
Good call!